spyplane/services/config_service.py

```diff
@@ -19,6 +21,25 @@ async def dump_config_embed(self) -> Embed:
             embed.add_field(name=config.name, value=config.value, inline=False)
         return embed
 
+    async def update_config(self, name: str, value: str):
+        # TODO refactor when we have more configuration
+        supported_configs = ["carryover", "interval_hours"]
+        if name not in supported_configs:
```

Comment:
You probably want to lower cast name first

Reply:
Done